.circleci/config.yml

```diff
@@ -312,6 +363,19 @@ workflows:
           <<: *filter-dev-branch
           requires:
             - deploy-staging
+      - notify-test-results:
+          name: notify-test-results-alpha
+          <<: *filter-dev-branch
+          requires:
+            - deploy-alpha
+          job_name: integration-tests-alpha
```

Comment:
`job_name` in workflow:
The name key can be used to invoke reusable jobs across any number of workflows. Using the name key ensures numbers are not appended to your job name (i.e. sayhello-1 , sayhello-2, etc.). The name you assign to the name key needs to be unique, otherwise the numbers will still be appended to the job name.